demos/segmentation_demo/main.cpp

```diff
@@ -2,29 +2,23 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include <gflags/gflags.h>
 #include <iostream>
-#include <memory>
-#include <map>
 #include <string>
 #include <vector>
-#include <iomanip>
 
-#include <inference_engine.hpp>
+#include <gflags/gflags.h>
 
-#include <format_reader_ptr.h>
+#include <opencv2/videoio.hpp>
 
+#include <inference_engine.hpp>
+
+#include <monitors/presenter.h>
 #include <samples/common.hpp>
+#include <samples/ocv_common.hpp>
 #include <samples/slog.hpp>
-#include <samples/args_helper.hpp>
-
-#include <vpu/vpu_tools_common.hpp>
-#include <vpu/vpu_plugin_config.hpp>
 
 #include "segmentation_demo.h"
 
-using namespace InferenceEngine;
```

Comment:
Could you not remove this, at least for now? It makes the diff a lot harder to read. Plus, we seem to have this declaration in every other demo, so it would be inconsistent to just remove it from here.

Reply:
Okay